touchforms/backend/test_persistence.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import with_statement
+import unittest
+import os
+from setup import init_classpath
+init_classpath()
+import persistence
+from setup import init_classpath
+
+CUR_DIR = os.path.dirname(__file__)
+
+class SqlitePersistenceTest(unittest.TestCase):
+
+    def setUp(self):
+        print "setup"
+
+    def test_sqlite_set(self):
+        user = "test_user"
+        persistence.postgres_set_sqlite(user, 1)
+        date_one = persistence.postgres_lookup_sqlite_last_modified(user)
+        persistence.postgres_set_sqlite(user, 2)
+        date_two = persistence.postgres_lookup_sqlite_last_modified(user)
+        version = persistence.postgres_lookup_sqlite_version(user)
+        assert(date_two > date_one)
```

Comment:
i think if you do `self.assertTrue()` it will properly tell you if the test failed or hit an error. in this case the test will always look like it hit an error https://docs.python.org/2/library/unittest.html#test-cases